### PR TITLE
Removed includes from common

### PR DESCRIPTION
**Pull request by bill-luu**

### CODE REVIEW

**Comment by ExtraGravity** [comment]
Please fix this build

**Comment by bill-luu** [comment]
@enochtsang Merge this if it looks good to you

File: src/telemTestTool.pro [hunk 1]
```diff
@@ -2,7 +2,7 @@
     error( "Couldn't find the common.pri file!" )
 }
 
-QT += widgets
+QT += widgets serialport
```
Comment (ExtraGravity) [inline src/telemTestTool.pro]: Stuff like this is what the common.pri file is for, did you try putting it there?

Comment (bill-luu) [reply src/telemTestTool.pro]: I remember taking it out for some reason, I think it made the travis build fail on the UCSetup Script? I'll try it again with the common in there first though.

**Pull request merged**